src/components/DebugMenu.js

```diff
@@ -123,6 +102,10 @@ const DebugMenu = () => {
     dispatch(routeTo(page));
   }
 
+  const onGetMessages = () => {
+    dispatch(fetchMessages());
+  }
+
   return (
     <View style={styles.container}>
```

Comment:
Please make the main view a ScrollView, since now you can't see all the new added actions